var/spack/repos/builtin/packages/font-util/package.py

```diff
@@ -70,12 +71,17 @@ class FontUtil(AutotoolsPackage, XorgPackage):
         f = f_r[0]
         resource(name=f, url=font_baseurl + f + '-' + f_r[1] + '.tar.gz',
                  sha256=f_r[2], destination=f, when='fonts=' + f)
+        conflicts('font=font-bh-tff', when='platform=cray')
```

Comment:
I'm okay with this change as is.  However, I expect this narrow conditional will need to be expanded eventually.  In fact, I needed this fix for `arch=linux-rhel7-broadwell`.

Reply:
Shouldn't this be:
```suggestion
        conflicts('fonts=font-bh-tff', when='platform=cray')
```
?